src/components/NavBar/NavigationBar.tsx

```diff
@@ -0,0 +1,179 @@
+"use client";
+
+import React from "react";
+import AppBar from "@mui/material/AppBar";
+import IconButton from "@mui/material/IconButton";
+import MenuIcon from "@mui/icons-material/Menu";
+import SwipeableDrawer from "@mui/material/SwipeableDrawer";
+import Link from "next/link";
+import { styled } from "styled-components";
+import Button from "@mui/material/Button";
+import LightDarkSlider from "./LightDarkSlider";
+
+import SignOutButton from "./SignOutButton";
+
+export const PageButton = styled(Button)`
+    color: white;
+    font-size: 1.25rem;
+    margin: 10px;
+    &:hover {
+        background-color: ${(props) => props.theme.primaryBackgroundColor};
+    }
+`;
+
+const UnstyledLink = styled(Link)`
+    text-decoration: none;
+    display: contents;
+`;
+
+const Logo = styled.img`
+    max-width: 100%;
```

Comment:
So width/height forces the image to stretch to be the size of the full box, however max-width and max-height do not do this. So in this case the image tries to be as big as it can, but is limited by the maximum height of the container. If the width was 100%, the image would still stretch to be the size of the container width wise, but because we only set the max width the image retains it's aspect ratio.